base64: clarify example descriptions that do not edit files inplace (#5656)
tldr-pages/tldr: pages/common/base64.md

> Encode or decode file or standard input to/from Base64, to standard output.
> More information: <https://www.gnu.org/software/coreutils/manual/html_node/base64-invocation.html>.

> - Encode the contents of a file as base64 to stdout:

Reviewer: Perhaps this would be more grammatically correct:
```suggestion
- Encode the contents of a file as base64 and write the result to stdout:
```

Author: Hmm, I get what you're saying, but that introduces the original issue again. I could read this as:

1. Encode the contents of a file in-place
2. Write the contents of the new file to stdout

I like it, but I think @bl-ue 's version is the most clear.

Reviewer: I'm not sure I quite understand how you're interpreting that, @makusu2. I would argue here that as it stands it is actually _more_ unclear now than if we add `and write the result to`, because we aren't indicating _where_ precisely the result is written to.

By specifically stating that we are writing the result to stdout, we avoid potential confusion that the file is being edited in place.

Author: Yeah, that's true. I was splitting the sentence on "and", but I agree now that it's more likely people will read it the way you're describing.

Maybe we could do 
```
Encode the contents of a file as base64, writing the result to stdout:
```

Hmm... I have no strong preference. 
But anyway, you're right, I'll apply the suggestion. Thanks!

> - Print a plaintext file's contents encoded in base64:

Reviewer 2: ```suggestion
- Encode the contents of a file as base64 to stdout:
```

Author: Words are hard, thanks for the suggestion!

`base64 {{filename}}`

> - Decode the base64 contents of a file to stdout:

Reviewer: Perhaps this would be more grammatically correct?
```suggestion
- Decode the base64 contents of a file and write the result to stdout:
```

> - Print a base64 encoded file's contents decoded to plaintext:

Reviewer 2: ```suggestion
- Decode the base64 contents of a file to stdout:
```
